src/Compilers/Core/Portable/Collections/TemporaryArray`1.cs

```diff
@@ -351,6 +351,58 @@ public void ReverseContents()
             }
         }
 
+        public void Sort(Comparison<T> compare)
+        {
+            if (_builder is not null)
+            {
+                _builder.Sort(compare);
+                return;
+            }
+
+            switch (_count)
```

Comment:
can you use: https://learn.microsoft.com/en-us/dotnet/api/system.memoryextensions.sort?view=net-8.0 instead?

Reply:
Basically:

```
            Span<T> span = MemoryMarshal.CreateSpan<T>(ref _item0, this.Count);
            span.Sort(comparison);
```

?

Reply:
Not available in netstandard2.0 :confused:

Reply:
Gah.  @stephentoub Any recommendations on how to best sort a span in netstandard2.0?